src/current/molt/molt-fetch.md

```diff
@@ -326,26 +345,23 @@ To customize the replication behavior (an advanced use case), use `--replicator-
 #### Replicate changes
 
 {{site.data.alerts.callout_info}}
-Before using this option, the source PostgreSQL or MySQL database **must** be configured for continuous replication, as described in [Setup](#replication-setup). MySQL 8.0 and later are supported.
+Before using this option, the source PostgreSQL or MySQL database **must** be configured for continuous replication, as described in [Setup](#replication-setup). MySQL 5.7 and later are supported.
 {{site.data.alerts.end}}
 
-`replication-only` instructs MOLT Fetch to replicate ongoing changes on the source to CockroachDB, using the specified replication marker.
+`replication-only` instructs MOLT Fetch to replicate ongoing changes on the source to CockroachDB, using the specified replication marker. This assumes you have already run [`--mode load-data`](#load-data) to load the source data into CockroachDB.
 
-- For a PostgreSQL source, first create a logical replication slot. For example, to create a replication slot named `replication_slot`:
-
-	{% include_cached copy-clipboard.html %}
-	~~~ sql
-	SELECT * FROM pg_create_logical_replication_slot('replication_slot', 'pgoutput');
-	~~~
-
-	In the `molt fetch` command, specify the replication slot name using `--pglogical-replication-slot-name`. For example:
+- For a PostgreSQL source, you should have already created a replication slot when [loading data](#load-data). Specify the same replication slot name using `--pglogical-replication-slot-name`. For example:
 
 	{% include_cached copy-clipboard.html %}
 	~~~
 	--mode replication-only 
 	--pglogical-replication-slot-name 'replication_slot'
 	~~~
 
+	{{site.data.alerts.callout_success}}
+	In case you want to run `replication-only` without already having loaded data (e.g., for testing), also include `--pglogical-publication-name` to specify a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) name and `--pglogical-publication-and-slot-drop-and-recreate` to ensure that the publication and replication slot are created in the correct order. For details on these flags, refer to [Global flags](#global-flags).
```

Comment:
Just a note that you technically do not need the `--pglogical-publication-name`. That's optional since we default to `molt_fetch`. But they should put the drop and recreate flag for sure in this mode.

This basically replaces the manual process.